core/nuxeo-operation.js

```diff
@@ -281,7 +329,7 @@ import './nuxeo-connection.js';
         .input(input)
         .execute(options);
 
-      if (this.async) {
+      if (this.async && !isBulk) {
```

Comment:
Well, in fact it shouldn't be the case. For example, the csv export button will continue to use the `async` adapter instead of the BAF automation endpoint. I think that this might be just poor naming here. Let me review it.